### PR TITLE
#travial support to RxSwift 6.x  (from 6.2 to 6.5 and latest)

### PR DESCRIPTION
**Pull request by ShenYj**
- [x] update RxSwift to 6.5 and latest [Swift Package Manager & CocoaPods]

### CODE REVIEW

**Review by ShenYj — COMMENTED** [review]
^_^

File: RxLocalizer.podspec [hunk 1]
```diff
@@ -10,8 +10,8 @@ s.license = { :type => "MIT", :file => "LICENSE" }
 s.author = { "Vladislav Khambir" => "vlad.khambir@gmail.com" }
 s.homepage = "https://github.com/RxSwiftCommunity/RxLocalizer"
 s.source = { :git => "https://github.com/RxSwiftCommunity/RxLocalizer.git", :tag => "#{s.version}" }
-s.dependency 'RxSwift', '~> 6.2.0'
-s.dependency 'RxCocoa', '~> 6.2.0'
+s.dependency 'RxSwift', '~> 6.0'
+s.dependency 'RxCocoa', '~> 6.0'
```
Comment (khambir) [inline RxLocalizer.podspec:14]: @ShenYj Why did you downgrade from 6.2 to 6.0?

**Review by khambir — CHANGES_REQUESTED** [review]
Why did you downgrade from 6.2 to 6.0 in podspec and package.swift?

File: RxLocalizer.podspec (hunk 1 above)
Comment (ShenYj) [reply RxLocalizer.podspec:14]: > @ShenYj Why did you downgrade from 6.2 to 6.0?

The previous version number was limited to the minor version, and will now be upgraded to 6.x

e.g. 6.5

**Pull request merged**